config/env/development.js

```diff
@@ -18,7 +18,8 @@ module.exports = {
 		}
 	},
 	app: {
-		title: 'MEAN.JS - Development Environment'
+		title: 'MEAN.JS - Development Environment',
+        livereload: true
```

Comment:
I don't have a compelling argument either way. Updated the spacing and moved it outside of app.